src/auth/dto/auth-credentials.dto.ts

```diff
@@ -1,11 +1,18 @@
-import { IsString, Matches, MaxLength, MinLength } from 'class-validator';
+import { IsString, Matches, MaxLength, MinLength, IsEmail } from 'class-validator';
 
 export class AuthCredentialsDto {
```

Comment:
Auth means login or signup, since we have now a `LoginCredentialsDto` can you please rename this to `SignUpCredentialsDto` and also its better to use `signIn` and `signUp` instead of `login` and `signUp` so please rename `LoginCredentialsDto` to `SignInCredentialsDto`